lib/pub_sub/active_record/events.rb

```diff
@@ -71,6 +74,21 @@ class AfterUpdateRollbackEvent < AfterRollbackEvent
 
       class AfterDestroyRollbackEvent < AfterRollbackEvent
       end
+
+      class ChangeEvent < PubSub::BaseEvent
+        def initialize(payload)
+          super({ timestamp: Time.now(), **payload })
```

Comment:
[here and elsewhere] = can you mass find/replace

Reply:
Oh, yeah, will do.